tests/functionaltests/Tests/NSURLConnection.mm

```diff
@@ -281,16 +281,14 @@ static void _testRequestWithURL(NSURLConnectionTestHelper* connectionTestHelper)
     LOG_INFO("Received HTTP response headers: %d", [httpResponse allHeaderFields]);
 
     // Make sure we received data.
-    ASSERT_EQ_MSG(NSURLConnectionDelegateDidReceiveData,
-                  [(NSNumber*)[connectionTestHelper.delegateCallOrder objectAtIndex:1] integerValue],
-                  "FAILED: didReceiveData should be the 2nd delegate to be called!");
+    ASSERT_TRUE_MSG([@(NSURLConnectionDelegateDidReceiveData) isEqual:[connectionTestHelper.delegateCallOrder objectAtIndex:1]],
```

Comment:
Oh, right. I hate those macros actually.

The way they're implemented, they'll break up the args to `ASSERT_OBJCEQ({@1, @2}, arr)` into `{@1`, `@2}` and `arr`, and then try to emit (simplified) `{@1 == @2}` (lel)